app/assets/javascripts/discourse/app/lib/logout.js

```diff
@@ -1,28 +1,14 @@
 import getURL from "discourse-common/lib/get-url";
 import { isEmpty } from "@ember/utils";
-import { findAll } from "discourse/models/login-method";
 import { helperContext } from "discourse-common/lib/helpers";
 
-export default function logout() {
+export default function logout({ redirect_url } = {}) {
```

Comment:
For javascript we prefer camelCase.

Reply:
This variable is coming direct from the server - used like `ajax("/logout").then(logout)`. So the snake case comes from rails.

But you're right having snake case in a function definition is weird... let me refactor this a little to make it tidier.